src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -502,19 +530,12 @@ export default class ClientMetricReport {
       observableVideoMetricSpec.dir
     );
     const metricSpec = metricMap[observableVideoMetricSpec.source];
-    const transform = metricSpec.transform;
-    const source = metricSpec.source;
-    const streamMetricReport = this.streamMetricReports[ssrcNum];
-    if (
-      streamMetricReport &&
-      observableVideoMetricSpec.source in streamMetricReport.currentMetrics
-    ) {
-      return source
-        ? transform(source, ssrcNum)
-        : transform(observableVideoMetricSpec.source, ssrcNum);
-    } else {
-      return source ? transform(source) : transform(observableVideoMetricSpec.source);
-    }
+    const { transform, source } = metricSpec;
+
+
+    return source
+      ? transform(source, ssrcNum)
+      : transform(observableVideoMetricSpec.source, ssrcNum);
```

Comment:
the video stream related metrics only exists in `streamMetricReport` and it requires `ssrc` to locate the corresponding `streamMetricReport`.
Remove the code path to get metric from `globalMetricReport` to reduce confusion.